microservices/volunteer.py

```diff
@@ -1,7 +1,95 @@
-# Create volunteer event (create volunteer event)
+# Flask application that query database
+# Flask application to query db
+import os
+from os import environ
 
-# Add volunteer (add user to volunteer table)
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
+from flask_cors import CORS
+
+# from datetime import datetime
+# import json
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = environ.get('dbURL') or 'mysql+mysqlconnector://root@localhost:3306/safeme'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+CORS(app)
+
+# Volunteer class with volunteerEventID, userID, userName, contact, timestamp with no ORM
+class Volunteer(db.Model):
+    __tablename__ = 'volunteers'
```

Comment:
Suggestion: rename table name to volunteer (aligns more with everything else)

Reply:
yes